apps/src/code-studio/components/ImagePicker.jsx

```diff
@@ -106,25 +127,40 @@ export default class ImagePicker extends React.Component {
       isStartMode = reduxState.level.isStartMode;
     }
 
-    const body =
-      !this.props.assetChosen || this.state.mode === 'files' ? (
-        <AssetManager
-          assetChosen={this.props.assetChosen}
-          assetsChanged={this.props.assetsChanged}
-          allowedExtensions={extensionFilter[this.props.typeFilter]}
-          uploadsEnabled={this.props.uploadsEnabled}
-          useFilesApi={this.props.useFilesApi}
-          projectId={this.props.projectId}
-          soundPlayer={this.props.soundPlayer}
-          disableAudioRecording={disableAudio}
-          imagePicker={true}
-          elementId={this.props.elementId}
-          levelName={levelName}
-          isStartMode={isStartMode}
-        />
-      ) : (
-        <IconLibrary assetChosen={this.getAssetNameWithPrefix} />
-      );
+    const getBody = () => {
```

Comment:
now that this is a function instead of a declaration, we should move it out of `render()` since it's getting pretty large